src/app/tl-header/tl-header.component.scss

```diff
@@ -179,6 +179,33 @@
       }
     }
 
+    wvr-nav-list-component[bottom-navigation] wvr-nav-li-component.supportTheLibraries {
+      --wvr-nav-link-background-hover: #355963;
+      width: fit-content;
+      font-family: var(--tl-font-family-sans-serif);
+      font-weight: 600;
+      font-size: 16px;
+      a {
+        margin: 0px 5px;
+        color: #355963;
+        text-decoration: none;
+        tl-icon-component svg {
+          fill: #355963;
+          width: 24px;
+          height: 65px;
+          margin-right: 5px;
+        }
+      }
+    }
+    wvr-nav-list-component[bottom-navigation] wvr-nav-li-component.supportTheLibraries:hover {
+      a {
+        color:#fff;
+        tl-icon-component svg {
+          fill: var(--tl-white);
```

Comment:
Please add a 2px right margin to the icon.

Reply:
tl-icon has a margin-right of 5px;